frontend/src/components/ChangelogModal/ChangelogModal.tsx

```diff
@@ -87,7 +87,7 @@ function ChangelogModal({ changelog, onClose }: Props): JSX.Element {
 
 	const onClickUpdateWorkspace = (): void => {
 		window.open(
-			'https://github.com/SigNoz/signoz/releases',
+			'https://signoz.io/docs/operate/migration/upgrade-standard',
```

Comment:
Can we point it to: https://signoz.io/docs/operate/upgrade/

Reply:
Updated url to https://signoz.io/upgrade-path/ based on discussion